clients/admin-ui/src/features/common/table/v2/PaginationBar.tsx

```diff
@@ -1,17 +1,29 @@
 import {
+  Button,
+  ChevronDownIcon,
   ChevronLeftIcon,
   ChevronRightIcon,
   HStack,
   IconButton,
+  Menu,
+  MenuButton,
+  MenuItem,
+  MenuList,
   Text,
 } from "@fidesui/react";
 import { Table as TableInstance } from "@tanstack/react-table";
 
+export const PageSizes = [25, 50, 100];
```

Comment:
can we all caps this? `PageSizes` makes me think it's a component when it's imported in `AddMultipleSystems.tsx`
```suggestion
export const PAGE_SIZES = [25, 50, 100];
```

Reply:
yes, good idea! https://github.com/ethyca/fides/pull/4351/commits/e0793ee2a8991fe9e1bc2609b9ab4e5435c49aa5